File: app/src/main/java/com/art2cat/dev/moonlightnote/model/NoteLab.java
package com.art2cat.dev.moonlightnote.model;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Rorschach on 12/17/16 5:10 PM.
 */

public class NoteLab {

  private List<Moonlight> MoonlightNote;

  public NoteLab() {
    MoonlightNote = new ArrayList<>();
  }

  public NoteLab(List<Moonlight> MoonlightNote) {
    this.MoonlightNote = MoonlightNote;
  }

  public List<Moonlight> getMoonlights() {
    return MoonlightNote;
  }

  public void setMoonlight(Moonlight moonlight) {
    MoonlightNote.add(moonlight);
  }

  public Moonlight getMoonlight(int i) {
    return MoonlightNote.get(i);
  }
}

File: app/src/main/java/com/art2cat/dev/moonlightnote/model/NoteLab.kt
package com.art2cat.dev.moonlightnote.model

import java.util.*

/**
 * Created by Rorschach
 * on 20/05/2017 11:36 PM.
 */

open class NoteLab {
    var MoonlightNote: MutableList<Moonlight>

    constructor() {
        MoonlightNote = ArrayList<Moonlight>()
    }

    constructor(MoonlightNote: MutableList<Moonlight>) {
        this.MoonlightNote = MoonlightNote
    }

    val moonlights: List<Moonlight>
        get() = MoonlightNote

    fun setMoonlight(moonlight: Moonlight) {
        MoonlightNote.add(moonlight)
    }

    fun getMoonlight(i: Int): Moonlight {
        return MoonlightNote.get(i)
    }
}